presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -28,6 +28,12 @@ Mathematical Functions
     The successProbability must be real value in [0, 1], numberOfTrials and value must be
     positive integers with numberOfTrials greater or equal to value.
 
+.. function:: cauchy_cdf(median, scale, value) -> double
+
+    Compute the Cauchy cdf with given parameters median and scale (gamma): P(N; median, scale).
```

Comment:
nits: just put these in normal paragraphs that wraps in reasonable length is fine. No need to put each sentence in a new line.